src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+// ACTIONS
+const ADD_BOOK = 'bookstore/books/ADD';
+const REMOVE_BOOK = 'bookstore/books/REMOVE';
```

Comment:
**[OPTIONAL]** Good job 👍 implementing the changes requested but the previous reviewer but there are a few things that can also be updated as suggested by the previous reviewer I highly recommend changing these two lines to something like this:

```
const ADD_BOOK = bookstore/books/ADD_BOOK
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```